Makefile

```diff
@@ -28,7 +28,7 @@ RUNTIME_IMAGE ?= gcr.io/distroless/static
 GIT_TAG ?= dirty-tag
 GIT_VERSION ?= $(shell git describe --tags --always --dirty)
 GIT_HASH ?= $(shell git rev-parse HEAD)
-DATE_FMT = +'%Y-%m-%dT%H:%M:%SZ'
+DATE_FMT = +%Y-%m-%dT%H:%M:%SZ
```

Comment:
I believe we have that format on rekor too https://github.com/sigstore/rekor/blob/main/Makefile#L33

Reply:
it sounds like it needs to be changed there too

Reply:
yep, was doing that :)

Reply:
Nice